messages/toolbar/toolbar.ar.yml

```diff
@@ -0,0 +1,4 @@
+kendo:
+  toolbar:
+    # The title for the more button
+    moreTitle: More tools
```

Comment:
I would go with something more descriptive like "The title of the "more tools" button in a responsive ToolBar", and `moreToolsTitle` as the property name